deep-learning/_posts/2017-08-31-egomotion.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: review
+title: "Unsupervised Learning of Depth and Ego-Motion from Video"
+tags: deep-learning CNN RNN
+author: "Faezeh Amjadi"
+cite:
+    authors: "Tinghui Zhou, Matthew Brown, Noah Snavely,David G. Lowe"
+    title:  "Unsupervised Learning of Depth and Ego-Motion from Video" 
+    venue:   "CVPR 2017"
+pdf: "https://people.eecs.berkeley.edu/~tinghuiz/projects/SfMLearner/cvpr17_sfm_final.pdf"
+---
+   
+
+
+
+## Prerequisite
+
+
+***novel view synthesis image***: given one input view of a scene, synthesize a new image of the scen seen from a different camera pose.
+If we have rotation, translation and intern camera matrix (R,T,K) **or** depth and translation matrix (D,T), would be enough to generate new view image.
+
+$$ x_{s} = K_{1} R_{1} T_{1} P_{w}$$<br>
+
+$$ x_{t} = \acute{ K_{1}} \acute{R_{2}} \acute{T_{2}} P_{w}$$<br>
+
+$$ x_{s} =K_{1} R_{1} T_{1} \acute{T_{2}}^{-1} \acute{R_{2}}^{-1} \acute{K_{1}}^{-1} x_{t} $$<br>
+
+
+***Forward/Backward Warping*** : There are two ways to get from a pixel’s original location in an image to its new location. The first method is forward warping. In this method, every point in the original image is transformed and sent to its new location. However, this mode of warping can result in holes and splattering. The better approach is to perform inverse warping. This algorithm goes through every pixel in the new, transformed image, undoes the transformation, and figures out which original pixel to grab. If the original pixel happens to fall between two pixels, simply interpolate the source image.
+
+![](/deep-learning/images/icstn/img0.png =200x200)
+<img src="/deep-learning/images/icstn/img0.png"  width="400" height='200' />
+
+**Note:**
+Some factors corrupt the process such as: The scene is dynamic or there is  occlusion between target and source images or surface is Lambertian.
+
+
+                             
+## Summary
+
+The authors present an unsupervised and end to end method for novel view synthesis image by CNN and also it does not need the pose information, that it would be a part of learning framework. The proposed network contains two parts: **depth prediction** network and **pose** network. The input of depth network is just target image and its output is depth prediction, also the input of pose network is source images and target images and depth predictions as shown below. 
+
+<img src="/deep-learning/images/icstn/img1.png"  width="600" height='300' />
```

Comment:
Please change the with and height of the image so it does not look distorted.  Same comment for the  other images